index.html

```diff
@@ -49,7 +49,7 @@
         <section class="head flex">
           <h1 class="head__title fz-40 fw-700 lh-52">I'm Joseph Glad to see you!</h1>
```

Comment:
- [ ] Great job so far 👍 Kindly make this heading to take up two lines just like in the template 
- Your design:
![Annotation 2022-03-24 215822](https://user-images.githubusercontent.com/85113567/160010137-e85c9a36-a187-4158-b27e-83efd7b9959c.jpg)
- Figma design:
![Annotation 2022-03-24 215834](https://user-images.githubusercontent.com/85113567/160010166-901215b4-e01d-41bd-8947-0e890f278b29.jpg)

Reply:
Done. Thanks a lot. @Kingstalux